### PR TITLE
Fix Indentation, Removed repetitive code

### PR DESCRIPTION
**Pull request by i-break-codes**
- Indentation fix in Infobox.js
- Added fall-through (i.e OR for key case, to avoid repetitive code) - For gradient navigation using left/right arrow keys

### CODE REVIEW

File: source/scripts/modules/Keys.js [hunk 1]
```diff
@@ -13,19 +13,12 @@ window.uiGradients = window.uiGradients || {};
...
-          break;
-
           case 39: // right
-            if (pallete.hasClass('is-active')) {
-              return;
-            }
-            canvas.trigger('changeUp');
+            if (pallete.hasClass('is-active')) return;
+
+            (event.which === 37 ) ? canvas.trigger('changeDown') : canvas.trigger('changeUp');
```
Comment (ghosh) [inline source/scripts/modules/Keys.js]: Hmmm...not sure we can do this. In the case when the left button is pressed, the execution will never get into this block.

Comment (ghosh) [inline source/scripts/modules/Keys.js]: Aah..ok. I see it now. That's clever. Thanks, I'll merge it soon.

Comment (i-break-codes) [inline source/scripts/modules/Keys.js]: No problem, its called [fallthrough](http://stackoverflow.com/questions/188461/), give it a read :)

Comment (ghosh) [inline source/scripts/modules/Keys.js]: After reading more about it, I was thinking maybe it is better to leave it as, for the sake of simplicity.

While I see the benefits, in this particular case having an if statement within kind of defeats the purpose. I would rather duplicate one line of code and preserve readability.

Anyway, thanks for introducing fallthrough to me, I am definitely going to use it to clean up some contrived if-else statements of mine. Live and learn!

**Comment by i-break-codes** [comment]
Yea kinda makes sense to move the dupe code in a function and calling it inside the switch statements.. no issues though! Glad to share something new with you ;)

**Pull request closed without merging**